protocol/0026-AUCT-auctions.md

```diff
@@ -157,24 +157,24 @@ message Market {
+- As the Vega network, in auction mode, all orders are placed in the book but never uncross until the end of the auction period. (<a name="0026-AUCT-004" href="#0026-AUCT-004">0026-AUCT-004</a>) for product spot: (<a name="0026-AUCT-024" href="#0026-AUCT-024">0026-AUCT-024</a>)
+- As a user, I can cancel an order that it either live on the order book or parked. (<a href="./0068-MATC-matching_engine.md#0068-MATC-033">0068-MATC-033</a>) for product spot: (<a href="./0068-MATC-matching_engine.md#0068-MATC-060">0068-MATC-060</a>)
+- As a user, I can get information about the trading mode of the market (through the [market framework](./0001-MKTF-market_framework.md)) (<a name="0026-AUCT-005" href="#0026-AUCT-005">0026-AUCT-005</a>)for product spot:(<a name="0026-AUCT-025" href="#0026-AUCT-025">0026-AUCT-025</a>)
+- As a user, I can get real time information through the API about a market in auction mode: indicative crossing price, indicative crossing volume.  (<a name="0026-AUCT-006" href="#0026-AUCT-006">0026-AUCT-006</a>) for product spot: (<a name="0026-AUCT-026" href="#0026-AUCT-026">0026-AUCT-026</a>)
+- As a user, the market depth API provides the same data that would be sent during continuous trading (<a name="0026-AUCT-007" href="#0026-AUCT-007">0026-AUCT-007</a>) for product spot: (<a name="0026-AUCT-027" href="#0026-AUCT-027">0026-AUCT-027</a>)
+- As an API user, I can identify: (<a name="0026-AUCT-008" href="#0026-AUCT-008">0026-AUCT-008</a>) for product spot: (<a name="0026-AUCT-028" href="#0026-AUCT-028">0026-AUCT-028</a>)
   - If a market is temporarily in an auction period
   - Why it is in that period (e.g. Auction at open, liquidity sourcing, price monitoring)
   - When the auction will next attempt to uncross or if the auction period ended and the auction cannot be resolved for whatever reason then this should come blank or otherwise indicating that the system doesn't know when the auction ought to end.
 - A market with default trading mode "continuous trading" will start with an opening auction. The opening auction will run from the close of voting on the market proposal (assumed to pass successfully) until:
-    1. the enactment time assuming there are orders crossing on the book and [liquidity is supplied](./0038-OLIQ-liquidity_provision_order_type.md). (<a name="0026-AUCT-017" href="#0026-AUCT-017">0026-AUCT-017</a>)
-    2. past the enactment time if there is no [liquidity supplied](./0038-OLIQ-liquidity_provision_order_type.md). The auction won't end until sufficient liquidity is committed. (<a name="0026-AUCT-018" href="#0026-AUCT-018">0026-AUCT-018</a>)
+    1. the enactment time assuming there are orders crossing on the book and [liquidity is supplied](./0038-OLIQ-liquidity_provision_order_type.md). (<a name="0026-AUCT-017" href="#0026-AUCT-017">0026-AUCT-017</a>) for product spot: (<a name="0026-AUCT-029" href="#0026-AUCT-029">0026-AUCT-029</a>)
```

Comment:
For Spot this seems to contradict this spec
https://github.com/vegaprotocol/specs/blob/cosmicelevator/protocol/0080-SPOT-product_builtin_spot.md

Reply:
Good point, updated it, thanks